wire-ios/Wire-iOS Tests/ConversationOptionsViewControllerTests.swift

```diff
@@ -71,11 +71,13 @@ final class ConversationOptionsViewControllerTests: XCTestCase {
     var mockConversation: MockConversation!
     var mockUserSession: UserSessionMock!
     var mockCreateSecuredGuestLinkUseCase: MockCreateConversationGuestLinkUseCaseProtocol!
+    private var snapshotHelper: SnapshotHelper!
```

Comment:
Should we take this opportunity to rename all cases and use "SnapshotTests" in all touched files? 🤔

Reply:
Already discussed 1:1 I will do at a later time.